README.md

```diff
@@ -88,10 +88,12 @@ This will stop and start the polling interval in order to renew polling with new
 
 ## Events
 
-The proxy client emits two different events you can subscribe to:
+The proxy client emits those different events you can subscribe to:
 
 - "ready"
 - "update"
+- "sent" (metrics sent)
+- "error" (metrics sending error)
```

Comment:
"sent" and "error" are the same event names we have in Node.js client that I use as a reference impl. Currently error event is only emitted on metrics error since swift SDK didn't event emit any other error events before